includes/admin/class-wc-payments-admin.php

```diff
@@ -279,6 +279,7 @@ public function register_payments_scripts() {
 				'isUpeSettingsPreviewEnabled' => WC_Payments_Features::is_upe_settings_preview_enabled(),
 				'isUpeEnabled'                => WC_Payments_Features::is_upe_enabled(),
 			],
+			'isSSLCheckoutEnforced'  => get_option( 'woocommerce_force_ssl_checkout', 'no' ),
```

Comment:
What do you think of passing the same value used in the `is_available` check here? We could extract the check to a new method and use it in both places. That way, the JS and PHP logic would be shared, the check to display the notice would be simpler, and we wouldn't need `isSiteUsingHttps`.

Reply:
Great idea! I've changed it for `isGatewayAvailable` and passed it down to display the task. Also removed the unused `isSiteUsingHttps` utility.